package.json

```diff
@@ -30,7 +30,7 @@
     "cookie-parser": "^1.4.6",
     "express": "^4.17.3",
     "express-validator": "^6.14.0",
-    "govuk-frontend": "^4.0.1",
+    "govuk-frontend": "^4.4.0",
```

Comment:
if we need to upgrade the govuk-frontend package, we also need to upgrade the assets the come with 4.4.0 in the CDN repository as it will be using new fonts, stylesheets etc..
https://github.com/companieshouse/cdn.ch.gov.uk/tree/master/app/assets/fonts/govuk-frontend
https://github.com/companieshouse/cdn.ch.gov.uk/tree/master/app/assets/javascripts/govuk-frontend
https://github.com/companieshouse/cdn.ch.gov.uk/tree/master/app/assets/images/govuk-frontend
https://github.com/companieshouse/cdn.ch.gov.uk/tree/master/app/assets/stylesheets/govuk-frontend

Reply:
reverted to 4.0.1

closed pull request in CDN repo (https://github.com/companieshouse/cdn.ch.gov.uk/tree/feature/roe-1584-assert-for-v-4-4-0)